packages/app-addresses/src/Editor.tsx

```diff
@@ -245,6 +235,4 @@ class Editor extends React.PureComponent<Props, State> {
   }
 }
 
-export default withObservableBase(
-  addressObservable.subject, { propName: 'addressAll' }
-)(translate(Editor));
+export default translate(Editor);
```

Comment:
What do you think about passing `t` down as a prop from index.tsx instead of wrapping this component with `translate` too?

Reply:
I really don't want to start passing t around everywhere. So would really want to stay away from `translate(App)` and then passing it along the tree.

So if we have `App -> B(t)` and only `B` actually have translatable text, it already is an issue. (Which is mostly the case everywhere) 

Now if we have `App -> B -> C(t)` and we don't use it in App and don't use is in B, it just becomes this stream of props.